src/dctkit/mesh/circumcenter.py

```diff
@@ -3,47 +3,49 @@
 import numpy.typing as npt
 from typing import Tuple
 
-# FIXME: pass only the coordinates of the nodes of the simplex or consider making this a
-# method of the SimplicialComplex class.
 
-
-def circumcenter(s: npt.NDArray,
+def circumcenter(S: npt.NDArray,
                  node_coords: npt.NDArray) -> Tuple[npt.NDArray, npt.NDArray]:
     """Compute the circumcenter of a given simplex. (Reference: Bell, Hirani,
         PyDEC: Software and Algorithms for Discretization of Exterior Calculus,
         2012, Section 10.1).
 
         Args:
-            s: array containing the IDs of the nodes beloging to the given simplex.
+            S: matrix containing the IDs of the nodes belonging to each simplex.
             node_coords: coordinates (cols) of each node of the complex to which the
-                given simplex belongs.
```

Comment:
When continuing a comment line within the list of arguments of a function docstring indentation is needed for proper formatting. Please fix here and **everywhere** else in the code.